packer/deploy-ami.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import sys
+import time
+
+import boto3
+
+logger = logging.getLogger(name=__name__)
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stderr)
+logger.addHandler(handler)
+
+yaml_template ='''
+    {}:
+      '64': {}
+'''.strip('\r\n')
+
+parser = argparse.ArgumentParser(description='Copy an image to all available regions')
+parser.add_argument('-r', '--region', default='us-east-1')
```

Comment:
should we just make these positional?

Reply:
I feel like making them positional is asking for trouble with the order